scalastyle-config.xml

```diff
@@ -203,19 +203,6 @@ This file is divided into 3 sections:
     ]]></customMessage>
  </check>
 
- <check customId="awaitresult" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
-  <parameters>
-   <parameter name="regex">Await\.result</parameter>
-  </parameters>
-  <customMessage><![CDATA[
-      Are you sure that you want to use Await.result? In most cases, you should use ThreadUtils.awaitResult instead.
-      If you must use Await.result, wrap the code block with
-      // scalastyle:off awaitresult
-      Await.result(...)
-      // scalastyle:on awaitresult
-    ]]></customMessage>
- </check>
-
```

Comment:
why this removed?

Reply:
this was giving error at random places, thus removed it.